app/views/planning_applications/validation/validation_requests/new.html.erb

```diff
@@ -0,0 +1 @@
+<%= render "planning_applications/validation/#{@type.underscore}s/new" %>
```

Comment:
Use the validation subclass itself to get the name, e.g.

``` irb
>> OtherChangeValidationRequest.model_name.plural
=> "other_change_validation_requests"
```

This should fix one of the brakeman errors